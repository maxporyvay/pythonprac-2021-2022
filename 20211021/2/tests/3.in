:add a b a+b
:sub a b a-b
add 2 3
add 4 16
sub 3 9
quit "{} functions, {} lines"