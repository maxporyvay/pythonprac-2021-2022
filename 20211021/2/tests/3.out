5
20
-6
3 functions, 6 lines
